### DISCUSSION TITLE
Custom data props for cells

### DISCUSSION THREAD

**Discussion by oe-bayram** [TanStack/table#4565] [General]
Is there a way to pass a second accessorFn to a column or generally pass custom data props to a cell?
I need it for this scenario: 
For displaying it to the user --> use the accessorFn which returns JSX.
But for exporting the data --> use another accessorFn or custom data props of the cell as string.
So I need the same data rendered in different ways. 
Example column:

```
columns: [
  {
    header: "CalculationOperations",
    accessorKey: ArticleListColumnDataField.ARBEITSGAENGE,
    accessorFn: (row) => (
      <CalculationOperationsColumnFormatter
        calculation={row.calculation}
      />
    ),
    enableSorting: false,
    cell: (info) => info.getValue(),
    footer: (props) => props.column.id,
    accessorRawFn: (row) =>
      utils.graphql
        .getDefinedItems(row.calculation?.calculationOperations?.items)
        .map(
          (calculationOperation) =>
            calculationOperation.operationType.name
        )
        .join(", "),
  },
  ...
```

**Comment by KevinVandy** [comment]
You should only use accessorFn for accessing data, not JSX. Use cell for wrapping JSX.

**Reply by oe-bayram** [reply]
And how could I use the accessorFn for exporting the data as .csv for example?
